Matveibos/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+
+# this class is Hash
+class MyHash
+  include Enumerable
+
+  attr_reader :pair_count, :size, :arr
+
+  LOAD_FACTOR = 0.5
+  SIZE = 10
+
+  def initialize
+    initialize_hash
+  end
+
+  def [](key)
+    ident = key.hash % @size
```

Comment:
`ident = key.hash % @size` is repeated 3 times in the code, lets move it to the separate method